OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -20,46 +20,157 @@ namespace OpenRA.Mods.Common.Widgets
 	public class SupportPowerTimerWidget : Widget
 	{
 		public readonly string Font = "Bold";
-		public readonly string Format = "{0}: {1}";
-		public readonly TimerOrder Order = TimerOrder.Descending;
+		public readonly string TimerFormat = "{0}: {1}";
```

Comment:
Unless there is some rule I don't know of, this value will never change. So yes, this should be const.